_includes/script.html

```diff
@@ -4,19 +4,13 @@
 <script type="text/javascript" id="hs-script-loader" async defer src="//js.hs-scripts.com/8551838.js"></script>
 <!-- End of HubSpot Embed Code -->
 
-{% if jekyll.environment == "production" %}
-<!-- This is installed from the https://github.com/slashdotdash/jekyll-lunr-js-search Gem -->
-<script src="/js/search.min.js" type="text/javascript" charset="utf-8"></script>
-{% endif %}
-
-<script type="text/javascript">
-    $(function() {
-      $('#search-input').lunrSearch({
-        indexUrl  : '/js/index.json',           // url for the .json file containing search index data
-        results   : '#search-results',          // selector for containing search results element
-        template  : '#search-results-template', // selector for Mustache.js template
-        titleMsg  : '', //<h1>Search results<h1>',   // message attached in front of results (can be empty)
-        emptyMsg  : '<p>Nothing found.</p>'     // shown message if search returns no results
-      });
-    });
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@docsearch/css@3" />
+<script src="https://cdn.jsdelivr.net/npm/@docsearch/js@3" ></script>
+<script>
+docsearch({
+  container: '#docsearch',
+  appId: '4SXF6GYFKZ',
+  apiKey: '2028276a4fc7f09eeb0bd082fbd85164',
```

Comment:
Should we rotate this and put it in a github secret, or is it not very sensitive?

Reply:
It's exposed in the frontend code in the generated site, not considered secret- it's a read-only key.